doc/kconfig/conf.py

```diff
@@ -1,65 +1,35 @@
 # Kconfig documentation build configuration file
 
-import os
+from pathlib import Path
+import sys
 
-NRF_BASE = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
 
-# -- General configuration ------------------------------------------------
+# Paths ------------------------------------------------------------------------
 
-# Sphinx 2.0 changes the default from 'index' to 'contents'
-master_doc = 'index'
+NRF_BASE = Path(__file__).absolute().parent / ".." / ".."
 
-# General information about the project.
-project = 'Kconfig reference'
-copyright = '2019-2021, Nordic Semiconductor'
-author = 'Nordic Semiconductor'
+sys.path.insert(0, str(NRF_BASE / "doc" / "utils"))
+import utils
 
-# Get rid of version number while keeping the spacing the same as for other
-# docsets
-version = '&nbsp;'
+# General configuration --------------------------------------------------------
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+project = "Kconfig reference"
+copyright = "2019-2021, Nordic Semiconductor"
+author = "Nordic Semiconductor"
+# NOTE: use blank space as version to preserve space
+version = "&nbsp;"
 
-# -- Options for HTML output ----------------------------------------------
+# Options for HTML output ------------------------------------------------------
 
-html_theme = "kconfig"
-html_theme_path = ['{}/doc/themes'.format(NRF_BASE)]
-html_favicon = '{}/doc/static/images/favicon.ico'.format(NRF_BASE)
-
-# The name for this set of Sphinx documents.  If None, it defaults to
-# "<project> v<release> documentation".
-html_title = "Kconfig reference"
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['{}/doc/static'.format(NRF_BASE)]
-
-# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
-# using the given strftime format.
-html_last_updated_fmt = '%b %d, %Y'
-
-# If false, no module index is generated.
-html_domain_indices = False
-
-# If false, no index is generated.
-html_use_index = True
-
-# If true, the index is split into individual pages for each letter.
-html_split_index = True
-
-# If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
+html_theme = "sphinx_ncs_theme"
+html_favicon = str(NRF_BASE / "doc" / "static" / "images" / "favicon.ico")
```

Comment:
That requires more customizations (layout file) to the default rtd theme. I think it is better to minimize theme customizations to ease maintenance for now.

Reply:
But that means we need to add files to every doc set.
I think we would be saving more work by putting it into the theme, but up to you.

Reply:
The alternative is to have a customized copy of https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/layout.html in the theme. It would likely allow us to fix duplicate JS, too.

Reply:
That would be good - we'll probably need to make changes to that one in the future anyway.

Reply:
Found a way to extend layout without holding a duplicate, so I have fixed both duplicate ncs.js and added favicon.